sync/linux/controlplane.sh

```diff
@@ -115,7 +107,7 @@ networking:
   podSubnet: "100.244.0.0/16"
 EOF
 
-sudo kubeadm init --config=/var/sync/shared/kubeadm.yaml --v=6
+sudo kubeadm init --config=/var/sync/shared/kubeadm.yaml --v=6 --ignore-preflight-errors=All
```

Comment:
Maybe add a comment why adding  --ignore-preflight-errors=All ? Is it because dockershim ? Swap Error?